glyphs-reader/src/font.rs

```diff
@@ -46,6 +46,7 @@ pub struct RawAxisUserToDesignMap(Vec<(OrderedFloat<f32>, OrderedFloat<f32>)>);
 #[derive(Debug, PartialEq, Hash)]
 pub struct Font {
     pub units_per_em: u16,
+    pub use_typo_metrics: Option<bool>,
```

Comment:
I don't know.. `use_typo_metrics` in second position in this struct definition looks a bit too prominent for such a relatively insignificant bit that's basically almost always set nowadays...

Reply:
Hm, I think you are right and the presence at the top of static metadata does send a message, the wrong one.

Reply:
Moved to StaticMetadata::misc.